### DISCUSSION TITLE
support source for json fields

### DISCUSSION THREAD

**Discussion by aminiun** [encode/django-rest-framework#9163] [Ideas & Suggestions]
Hi!
Any plan to support `source` arg for json fields ?

For example:
```python
class TestSerializer(ModelSerializer):
    test = serializers.CharField(source="json_field.test")
    ...
```

**Comment by auvipy** [comment]
may I know from which version django support that?

**Reply by aminiun** [reply]
What Django support is filtering on json field, like `json_field__test`.

What I am suggesting is that to handle in DRF, for example by checking field type, then if the field type was sth like JsonField, then `getitem("test")`.

It can be really usefull when using `JsonField`.

**Reply by auvipy** [reply]
are you up for a contribution?

**Reply by aminiun** [reply]
Yeah why not.
I have had a small commit before.

**Reply by aminiun** [reply]
Any result for this conversion ?
